api/lib/controllers/meeting.js

```diff
@@ -199,6 +199,72 @@ module.exports = {
     const participants = await Participant.find({ meeting_id: _id });
 
     return res.status( 200 ).send( participants );
+  },
+
+  async getAllMeetings( req, res ) {
+    const subject_id = req.credentials.sub;
```

Comment:
`req.credentials` also includes `email` if I remember correctly which should make fetching participants much easier.

Reply:
Omg ty 🙏